packages/terra-form-input/src/InputField.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Field from 'terra-form-field';
+import Input from './Input';
+
+const propTypes = {
+  /**
+   * The defaultValue of the input field. Use this to create an uncontrolled input.
+   */
+  defaultValue: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+  ]),
+  /**
+   * Error message for when the input is invalid.
+   */
+  error: PropTypes.node,
+  /**
+   * Help element to display with the input.
+   */
+  help: PropTypes.node,
+  /**
+   * Whether or not to hide the required indicator on the label.
+   */
+  hideRequired: PropTypes.bool,
+  /**
+   * Attributes to attach to the input object
+   */
+  inputAttrs: PropTypes.object,
+  /**
+   * Id of the input. Populates both the input and the htmlFor prop of the wrapper Field.
+   */
+  inputId: PropTypes.string,
```

Comment:
What are people's thoughts on having this be required? This would ensure that we have an accessible link from the label to the input when people use this component.

Reply:
I think that's a good idea. Small changes like this greatly help with ensuring Terra is as accessible as it can be.